docs/docs/guides/evaluation/guardrails_and_monitors.md

```diff
@@ -5,139 +5,482 @@ import TabItem from '@theme/TabItem';
 
 ![Feedback](./../../../static/img/guardrails_scorers.png)
 
-Weave provides a robust framework for implementing safety controls and monitoring systems in LLM applications through a unified scoring system. This guide explains how to leverage scorers as both guardrails for active intervention and monitors for passive evaluation in production environments.
+## Introduction
```

Comment:
optional - Having both an Introduction and an Overview feels a little strange to me